bentoml/bundler/loader.py

```diff
@@ -107,6 +107,7 @@ def load_bento_service_class(bundle_path):
 
     # Prepend bundle_path to sys.path for loading extra python dependencies
     sys.path.insert(0, bundle_path)
```

Comment:
keeping this line for pip installed BentoService to load python dependencies properly